### PR TITLE
 add showing ActiveEon portals pragmas

### PR DESCRIPTION
**Pull request by mklkun**

### CODE REVIEW

File: proactive-jupyter-kernel/kernel.py [hunk 1]
```diff
@@ -191,10 +198,46 @@ def __trigger_pragma__(self, pragma_info):
...
+            data = IFrame(url, width=input_data['width'], height=input_data['height'])
+        else:
+            data = IFrame(url, width=1200, height=750)
+
+        content = {'data': {'text/html': data._repr_html_()},
+                   'metadata': {}
+                   }
+        self.send_response(self.iopub_socket, 'display_data', content)
+
+    def __show_scheduling_portal__(self, input_data):
```
Comment (Mykhailenko) [inline proactive-jupyter-kernel/kernel.py]: I wonder if we can factorize `__show_resource_manager__` and `__show_scheduling_portal__`

File: proactive-jupyter-kernel/pragma.py [hunk 2]
```diff
@@ -356,10 +374,36 @@ def is_valid_get_result(data):
...
+        raise ParsingError('Missing one of height/width parameters')
+    if 'width' in data and 'height' in data and \
+            not (re.match(pattern_dimension, data['width']) and re.match(pattern_dimension, data['height'])):
+        raise ParameterError('Invalid height/width parameters')
+    if 'host' in data and not re.match(pattern_path_cars, data['host']):
+        raise ParameterError('Invalid host parameter')
+    return
+
+
+def is_valid_show_scheduling_portal(data):
```
Comment (Mykhailenko) [inline proactive-jupyter-kernel/pragma.py]: Also if we can factorize `is_valid_show_resource_manager` and `is_valid_show_scheduling_portal`

File: proactive-jupyter-kernel/kernel.py [hunk 3]
```diff
@@ -133,6 +135,11 @@ def __start_proactive__(self):
 
         proactive_host = 'try.activeeon.com'
         proactive_port = '8080'
+
+        self.proactive_config['proactive_server'] = {}
```
Comment (Mykhailenko) [inline proactive-jupyter-kernel/kernel.py]: Can we have a constant somewhere for 'proactive_server'? I dont know it it is possible in Python tho

File: proactive-jupyter-kernel/kernel.py (hunk 1 above)
Comment (mklkun) [reply proactive-jupyter-kernel/kernel.py]: Yes! Sure. I will do it before merging.

File: proactive-jupyter-kernel/pragma.py (hunk 2 above)
Comment (mklkun) [reply proactive-jupyter-kernel/pragma.py]: Same!

File: proactive-jupyter-kernel/kernel.py (hunk 3 above)
Comment (mklkun) [reply proactive-jupyter-kernel/kernel.py]: Well, before your question I thought that the answer was no, but I checked to be sure and I found a way to do it. An "ugly" way, but a way lol. I will see if it is a good idea to adapt it.

**Pull request merged**